### DISCUSSION TITLE
1.1.0

### DISCUSSION THREAD

**Discussion by tim-hub** [tim-hub/obsidian-bible-reference#38] [Announcements]
- fix issue https://github.com/tim-hub/obsidian-bible-reference/issues/37
- refactor to clean up code base
**Full Changelog**: https://github.com/tim-hub/obsidian-bible-reference/compare/1.0.2...1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/tim-hub/obsidian-bible-reference/releases/tag/1.1.0'>1.1.0</a>.</em>

**Comment by zkaizen** [comment]
Hello 

Thank you for the work you are doing to make it easy to access the Bible in Obsidian.  It works well on my Macbook Air running the latest Mac OS, but somehow it does not give me the bible reference on my iPad OS running the latest OS and the most update Obsidian version on iPad OS v1.2.3(57) API v0.14.15.  Thanks for your help.

**Comment by tim-hub** [comment]
> Hello 
> 
> Thank you for the work you are doing to make it easy to access the Bible in Obsidian.  It works well on my Macbook Air running the latest Mac OS, but somehow it does not give me the bible reference on my iPad OS running the latest OS and the most update Obsidian version on iPad OS v1.2.3(57) API v0.14.15.  Thanks for your help.


Thanks for the feedback,

It seems like it is not possible to enable debug console in iOS obsidian.

https://forum.obsidian.md/t/access-the-obsidian-console-on-ios-app/23476/3

Sorry for that, and I don't have an iOS device for now.

 I don't have an idea about how to solve it now. Sorry for that.
